Gemfile

```diff
@@ -54,24 +53,26 @@ source "https://gems.contribsys.com/" do
   gem "sidekiq-pro"
 end
 
+gem 'acts-as-taggable-on'
 gem 'bixby', '~> 5.0'
-gem 'blacklight', '~> 7.18', '< 7.25'
-gem 'blacklight-gallery'
+gem 'blacklight', '~> 7.18'
+gem 'blacklight-gallery', '~> 4.1.0'
 gem 'blacklight-oembed'
-gem 'blacklight-spotlight', '~> 3.0'
+gem 'blacklight-spotlight', '3.6.0.beta10'
```

Comment:
FWIW, Spotlight `3.6.0.beta` was rebranded as `4.0` because of the breaking changes introduced.